dockers/tests/testMultiRelayer/r

```diff
@@ -0,0 +1,2 @@
+docker run --rm -ti -v /var/run/docker.sock:/var/run/docker.sock -e ROOT=$PWD -v $PWD/.env:/.env opengsn/relaydc:test2 "$@"
```

Comment:
Please rename 'r' to something that is more than one letter.

Reply:
done. its now "testrdc", as its test-only wrapper for rdc